worksheet_B/Mandelbrot.cpp

```diff
@@ -2,6 +2,126 @@
 //
 
 #include "stdafx.h"
+// HSV to RGB 
+// Source: "http://stackoverflow.com/questions/3018313/algorithm-to-convert-rgb-to-hsv-and-hsv-to-rgb-in-range-0-255-for-both"
+typedef struct {
+	double r;       // percent
+	double g;       // percent
+	double b;       // percent
+} rgb;
```

Comment:
Note that `typedef struct { ... } rgb` is a hangover from C -- in C++ it is more common to write `struct rgb { ... }`